MAUI/DataForm/getting-started.md

```diff
@@ -435,3 +435,32 @@ RadioGroup
 </td>
 </tr>
 </table>
+
+## Label view customization
+
+Customize the label view of default layout by using the `InitializeDataLabel` method of the [DataFormItemManager](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItemManager.html).
```

Comment:
Customize the label view of the default layout by using the `InitializeDataLabel` method of the [DataFormItemManager](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItemManager.html).

Reply:
Updated